taccsite_cms/templates/djangocms_forms/email_template/email.html

```diff
@@ -0,0 +1,19 @@
+{% load i18n djangocms_forms_tags %}<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+    <head>
+        <meta name="viewport" content="width=device-width" />
+        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+        <title>{{ title }}</title>
+    </head>
+    <body itemscope itemtype="http://schema.org/EmailMessage">
+        <h1>{{ form.title }}</h1>
+        <p><strong>{% trans 'Date:' %}</strong> {% now 'jS F Y H:i' %}</p>
+        <dl>
+        {% for item in form_data %}
+            <dt>{% if item.label %}{{ item.label }}{% else %}{{ item.name }}{% endif %}:</dt>
+            <dd>{{ item.value|friendly|default_if_none:'—' }}</dd>
+        {% endfor %}
+        </dl>
+        {% trans 'Sent via' %} <a href="http://{{ request.get_host }}{{ referrer }}">{{ request.get_host }}{{ referrer }}</a>
+    </body>
+</html>
```

Comment:
When someone wants to customize the e-mail, I don't want me to re-learn, or anyone to need to learn, how.

**But**, I _am_ willing to delete this file.

Reply:
Ooh, I got it. Replace this with a `.../email_template/email.html.md` file.[^1]

[^1]: Like you did for `.../form_template/default.html.md`.

Reply:
No. Nothing to tell others about. I've deleted the template: f3178f6.